packages/@lwc/engine-server/src/__tests__/fixtures/slots-light-evil-template-as-prop/modules/x/outer/outer.js

```diff
@@ -0,0 +1,12 @@
+import { LightningElement } from 'lwc';
+import innerTemplateFn from './innerTemplateFn.html';
+import tmpl from './outer.html';
```

Comment:
Does the behavior change if this is just a regular/implicit template, rather than being imported/rendered?